TFT/src/User/API/MachineParameters.c

```diff
@@ -37,6 +40,9 @@ const char * const parameterCode[PARAMETERS_COUNT] = {
   "M207",  // FW retract
   "M208",  // FW retract recover
   "M209",  // Set auto FW retract
+  "M665",  // Delta Configuration
```

Comment:
Param are ordered by gcode. So your gcode must be provided after M420. That logic is applied on all the files e.g. Languages, parseAck, inerfaceCmd etc._

Reply:
So M200 and M569 have wrong place no ? Problem if I put my gcode after M420, settings are not correctly applied, I have tested.

Reply:
Yes it was put in the wrong place while the others were properly ordered. I will clean up it in a further minor PR. If one of the new gcode conflicts with m420 or other gcodes it is better to identify the cause and fixing it (if possible) in parseAck only or to provide a comment highlighting the reason of different order. Also  verify that m420 continues to be properly handled if put after your gcodes. In any case the order in all the files (maybe with the exception in parseAck) can be provided

Reply:
@Guilouz Ok, but what is the entire ack message returned for M665 and M666? So we can understand the reason why M420 must be parsed after M665 and M666 (in parseAck)